### ISSUE TITLE
Display of Suggested answers for customised templates

### ISSUE THREAD

**Issue by vyruss** [DMPRoadmap/roadmap#277]
For customised Funder templates, the code right now performs a semi-random selection of which suggested answer to display (first result returned by the database).

For v0.3 we will display the customiser's suggested answer (and label it as such) if it exists, and if not, the original funder template's suggested answer (and label it as such). The display area should be no more than 7 lines long and scrollable.

For MVP-v0.4 we can make this into a tabbed display area, enabling selection between funder/original and customiser suggested answer.

**Comment by xsrust** [comment]
current text reads " #{org.abbreviation} Example of answer"
please comment during UAT if the text needs tweaked/changed.

text on app/templates/phases/_answer_form.html.erb ln 40

**Comment by xsrust** [comment]
https://github.com/DigitalCurationCentre/roadmap/commit/418fddbc8691df85fedae63ca3c22ed23b7e5984

**Comment by xsrust** [comment]
Addressed in PR: Annotations

**Comment by sjDCC** [comment]
@xsrust it would be better if this read "#{org.abbreviation} example answer" It doesn't really make sense to say 'Example of answer' so I've always been meaning to get this changed ;-)

This appears to still need some work. See tickets #381 #372 #334 and #270 These perhaps need merging

**Comment by xsrust** [comment]
sure, I can update the display to read that.
Since this ticket is just about the display, I think we should fix it separately to the others, which seem to be about data issues, either through migration, or impropper selection

**Comment by dsisu** [comment]
In an ideal world, we should get rid of the difference between example and suggested answers. The org_admin interface doesn't make it clear what the difference is.
![suggestedvsexampleanswer](https://cloud.githubusercontent.com/assets/2138956/26555309/602393d4-448c-11e7-87c8-a691037bb4f8.jpg)

**Comment by vyruss** [comment]
@dsisu where did you find this? This shouldn't exist in Roadmap as the difference has been eliminated

**Comment by dsisu** [comment]
Sorry, my fault, it is in the live version not dmponline-test. On dmponline-test you have 'default answer' and 'example answer'.

**Comment by sjDCC** [comment]
This has been changed so closing out.

A note for @dsisu, some example answers show without org names in the scrrengrab below. This is becauase the organisation doesn't have anything listed under 'abbreviation'. I've entered data for the few I've spotted but if we get helpdesk tickets, advise them to enter an abbreviation under their org details
![unnamed-example-answer](https://cloud.githubusercontent.com/assets/6169940/26645210/3e8edf9a-462f-11e7-8c1e-b54778922aa2.PNG)
